fastlane/lib/fastlane/setup/setup_ios.rb

```diff
@@ -276,8 +276,8 @@ def generate_fastfile(manually: false)
       end
 
       template.gsub!('snapshot', '# snapshot') unless self.tools[:snapshot]
-      template.gsub!('cocoapods', '') unless self.tools[:cocoapods]
-      template.gsub!('carthage', '') unless self.tools[:carthage]
+      template.gsub!('    cocoapods', '') unless self.tools[:cocoapods]
+      template.gsub!('    carthage', '') unless self.tools[:carthage]
```

Comment:
I think adding `# ` would be better like snapshot. @KrauseFx what do you think?

Reply:
Oh yeah, I'd be ok with adding a `#` instead of removing it 👍  Thanks y'all :)